_config.yml

```diff
@@ -29,6 +29,8 @@ imageNames:
   pilot-webhook: quay.io/calico/pilot-webhook
   flexvol: quay.io/calico/pod2daemon-flexvol
 
+include:
```

Comment:
is this a jekyll primitive? Or a netlify primitive? A comment here explaining would be nice

Reply:
Yes, it's a jekyll primitive. It's documented in the link in the comment at the top of the file.
It would be weird to add a comment just for include. It's right next to `exclude` and that's never had a comment